lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -128,8 +129,8 @@ class ScreenshotThumbnails extends Audit {
       }
       thumbnails.push({
         timing: Math.round(targetTimestamp - speedline.beginning),
-        timestamp: targetTimestamp * 1000,
-        data: base64Data,
+        timestamp: targetTimestamp,
```

Comment:
for traceOfTab we use microseconds for `timestamp`, I think we should keep matching it here (so `final-screenshot.js` will have to move to * 1000